server/src/main/java/com/nukkitx/server/network/bedrock/session/BedrockSession.java

```diff
@@ -327,7 +327,7 @@ public PlayerSession initializePlayerSession(NukkitLevel level) {
             playerSession = new PlayerSession(this, level);
         }
         if (!server.getSessionManager().add(this)) {
-            throw new IllegalStateException("Player could not be added to SessionManager");
+            throw new IllegalStateException("This player is already online.");
```

Comment:
Just because its already in the session's list, it doesn't mean he's online. It could be a bug or fault in the system.

Reply:
@astei